src/cli/manager.rs

```diff
@@ -73,6 +73,10 @@ struct Args {
     /// Enable webrtc thread test with limit of child tasks (can use port for webdriver as parameter).
     #[arg(long, value_name = "PORT", num_args = 0..=1, default_missing_value = "9515")]
     enable_webrtc_task_test: Option<u16>,
+
+    /// Sets the MAVLink System ID.
+    #[arg(long, value_name = "SYSTEM_ID", default_value = "1")]
+    mavlink_system_id: u8,
```

Comment:
@patrickelectric is all u8 range valid, or should we do some validation?

Reply:
I see no reason to not use the full range.